recipes/glimpse-bio/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "v1.1.1" %}
+
+package:
+  name: glimpse-bio
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/odelaneau/GLIMPSE/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ae3b4349c02f589022d5f22083f81e7a82dc93daf8c29d9beeeb7f8d34f80465
+  patches:
+    - 0001-fix-string_utils.h-warning.patch
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - htslib
+    - bzip2
+    - zlib
+    - boost
+    - boost-cpp
+    - openssl
+    - pthread-stubs
+  run:
+    - htslib
+    - bzip2
+    - zlib
+    - boost
+    - boost-cpp
```

Comment:
Is boost-cpp enough? or do you need for example the python bindings as well?

Reply:
There is no Python based on my understanding.

Reply:
then you can probably remove `boost`. `boost` includes additional dependencies __and__ `boost-cpp`. So I assume you only need boost-cpp.

Reply:
Done